src/components/Repo/RepoDetails.jsx

```diff
@@ -242,7 +242,29 @@ function RepoDetails() {
       : `Timestamp N/A`;
     return lastDate;
   };
-
+  const handleDeleteRepo = () => {
+    const confirmed = window.confirm('Are you sure you want to delete this repo?');
```

Comment:
We implemend dialogs using MUI's dialog component, not the native window API, check FilterDialog used in the ExplorePage for example.